README.md

```diff
@@ -1,24 +1,27 @@
 # Pong-3.0
-Pong 3.0 is colorful and fast-paced approach to the classic pong game. Play online with your friends and overwhelm them with your superior skills and balls!
+
+Pong 3.0 is a colorful and fast-paced approach to the classic pong game. Play online with your friends and overwhelm them with your superior skills and balls!
 
 ### Setting up
- * Clone the repo: `git clone ...`
- * Install the dependencies: `npm install`
+
+- Clone the repo: `git clone ...`
+- Install the dependencies: `npm install`
 
 ### Launch
- * `npm run dev` will start a process to watch for changes and rebuild the client automatically.
-It will also start the server on port 3000. You can now access the game at http://localhost:3000
 
-* At the moment, the server does not restart automatically when you change files, you'll have top restart it manually.
+- `npm run dev` will start a process to watch for changes and rebuild the client automatically.
+  It will also start the server on port 3000. You can now access the game at http://localhost:3000
 
+- At the moment, the server does not restart automatically when you change files, you'll have to restart it manually.
 
 ##### 💡 Tip
+
 > You can change the port on which the server will listen by setting the PORT environment variable
 >
 > e.g.: `PORT=42069 npm start`
 
-#### The technologies used for this project are : 
- * [Parcel](https://parceljs.org/) for bundling and building
- * [Socket.io](https://socket.io/) for websocket communication
- * [Pixi.js](https://www.pixijs.com/) for graphics
- 
+#### The technologies used for this project are :
+
+- [Parcel](https://parceljs.org/) for bundling and building
```

Comment:
It seems you've used some kind of automatic formatting, while the new formatting is fine, there is nothing wrong with the current one, and this pollutes the commit history. Please revert all changes that are only formatting changes.